wgpu-hal/src/vulkan/mod.rs

```diff
@@ -355,6 +355,11 @@ struct Swapchain {
     /// index as the image index, but we need to specify the semaphore as an argument
     /// to the acquire_next_image function which is what tells us which image to use.
     next_semaphore_index: usize,
+    /// The times which will be set in the next present times.
```

Comment:
```suggestion
    /// The times which will be set in the next present times.
```

"the times ... next present times" -> this doesn't sound very clear to me. How about just copying the upstream `The earliest time image should be presented` documentation?

Also, you use the plural `times` everywhere when `VkPresentTimeGOOGLE` (as the singular `time` in the name implies) only contains a single name.